tests/tacacs/conftest.py

```diff
@@ -15,19 +15,18 @@ def tacacs_creds(creds_all_duts):
     creds_all_duts.update(hardcoded_creds)
     return creds_all_duts
 
-
 @pytest.fixture(scope="module")
 def check_tacacs(ptfhost, duthosts, enum_rand_one_per_hwsku_hostname, tacacs_creds):
     logger.info('tacacs_creds: {}'.format(str(tacacs_creds)))
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
     tacacs_server_ip = ptfhost.mgmt_ip
-    setup_tacacs_client(duthost, tacacs_creds, tacacs_server_ip)
+    default_tacacs_server = setup_tacacs_client(duthost, tacacs_creds, tacacs_server_ip)
```

Comment:
Function `setup_tacacs_client` returns a list of servers, it would be better to use variable name `default_tacacs_servers`.